cluster-autoscaler/cloudprovider/aws/aws_util.go

```diff
@@ -53,6 +54,13 @@ type productAttributes struct {
 
 // GenerateEC2InstanceTypes returns a map of ec2 resources
 func GenerateEC2InstanceTypes(region string) (map[string]*InstanceType, error) {
+	var pricingUrlTemplate string
+	if strings.HasPrefix(region, "cn-") {
```

Comment:
minor: I am ok with this now. It would be better to use AWS SDK to check partition